README.md

```diff
@@ -1,54 +1,234 @@
 <sup>1 </sup>[MIT CSAIL](https://www.csail.mit.edu/),
 <sup>2 </sup>[Institute of AI and Fundamental Interactions (IAIFI)](https://iaifi.org/)<br>
-<sup>* </sup>Indicates equal contribution
+<sup>* </sup>Indicates equal contribution<br>
+CoRL 2023 (Oral)
+
+-----
+
+**Table of Contents**
+
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Downloading Example Datasets](#downloading-example-datasets)
+    - [Training a Feature Field](#training-a-feature-field)
+    - [Using our Custom Viewer](#using-our-custom-viewer)
+    - [Using the Nerfstudio Viewer](#using-the-nerfstudio-viewer)
+    - [Extracting CLIP and DINO Features](#extracting-clip-and-dino-features)
+- [Troubleshooting](#troubleshooting)
+- [Citation](#citation)
 
 ## Code
-**The NeRF and feature distillation code will be released soon.**
 
-We currently provide our implementation for extracting CLIP and DINO features. See [Usage](#usage) for more details.
+We currently release our implementation for training feature fields. We will release our custom viewer and the robot
+manipulation code soon.
+
+F3RM is built on top of [Nerfstudio](https://github.com/nerfstudio-project/nerfstudio) following their
+[guide for adding new methods](https://docs.nerf.studio/en/latest/developer_guides/new_methods.html). For a summary
+of the codebase structure, see [assets/code_structure.md](assets/code_structure.md).
 
 ### Installation
-**Note:** this repo will eventually require an NVIDIA GPU with CUDA 11.7+ for NeRF and feature field distillation.
 
-```bash
-# Clone the repo
-git clone https://github.com/f3rm/f3rm.git
-cd f3rm
+**Note:** this repo requires an NVIDIA GPU with CUDA 11.7+ for NeRF and feature field distillation.
+
+#### 1. Setup conda environment
 
-# Create conda environment. Feel free to use a different package manager
-conda create -n f3rm python=3.10
+```bash
+# We recommend that you use conda to manage your environment
+conda create -n f3rm python=3.8
 conda activate f3rm
+```
 
+#### 2. Install Nerfstudio dependencies
+
+```bash
 # Install torch per instructions here: https://pytorch.org/get-started/locally/
-# Choose the CUDA version that matches your GPU
+# Choose the CUDA version that your GPU supports. We will use CUDA 11.8
 pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 --extra-index-url https://download.pytorch.org/whl/cu118
 
-# Install F3RM project and dependencies
+# Install CUDA toolkit, you can skip this if you already have CUDA 11.8 installed
+# You can check your existing CUDA installation with `nvcc --version`
+conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit
+export CUDA_HOME=$CONDA_PREFIX
+
+# Install tiny-cuda-nn, this will take a few minutes
+pip install ninja git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
+```
+
+#### 3. Clone and install F3RM
+
+```bash
+git clone https://github.com/f3rm/f3rm.git
+cd f3rm
+
+# Install F3RM and its dependencies
 pip install -e .
```

Comment:
alternative is to add an installation option to install from git.

Reply:
When the robot stuff is implemented, this codebase will get more complicated and I don't think installing from git would be as suitable.